lib/openstax/rescue_from/configuration.rb

```diff
@@ -1,11 +1,29 @@
+require 'exception_notification'
+
 module OpenStax
   module RescueFrom
     class Configuration
 
-      # attr_accessor, attr_reader, attr_writers for variables
+      attr_accessor :raise_exceptions, :notifier, :html_error_template_path,
+        :html_error_template_layout_name, :app_name, :app_env, :email_prefix,
+        :sender_address, :exception_recipients
 
       def initialize
-        # set default values for variables
+        @raise_exceptions = ENV['RAISE_EXCEPTIONS'] || false
+
+        @app_name = ENV['APP_NAME'] || 'Tutor'
+        @app_env = ENV['APP_ENV'] || 'DEV'
```

Comment:
would also prefer no default env, and if not present don't use it.  'DEV' could easily be confused with our 'dev' which is a first stage production-like environment